docs/expectation_gallery/azure-pipelines-expectation-gallery.yml

```diff
@@ -173,65 +601,164 @@ stages:
+#           workingDirectory: $(Build.SourcesDirectory)/assets/scripts/
+#           displayName: 'Show testing errors'
+#
+#         - bash: cat gallery-tracebacks.txt
+#           workingDirectory: $(Build.SourcesDirectory)/assets/scripts/
+#           displayName: 'Show gallery tracebacks'
+#
+#         - task: S3Upload@1
+#           inputs:
+#             regionName: 'us-east-2'
+#             awsCredentials: 'aws-ci-great-expectations'
+#             bucketName: 'superconductive-public'
+#             sourceFolder: '$(Build.SourcesDirectory)/assets/scripts'
+#             globExpressions: '*.json'
+#             targetFolder: 'static/gallery/'
+#             filesAcl: 'public-read'
+
+
+  - stage: build_gallery_staging
+    dependsOn: exp_tests_on_all_backends
+    pool:
+      vmImage: 'ubuntu-20.04'
+
+    jobs:
+      - job: combine_backend_results
+        timeoutInMinutes: 180
+        variables:
+          python.version: '3.8'
+
+        steps:
+          - task: UsePythonVersion@0
+            inputs:
+              versionSpec: '$(python.version)'
+            displayName: 'Use Python $(python.version)'
+
+          - bash: python -m pip install --upgrade pip
+            displayName: 'Update pip'
+
+          - script: |
+              pip install \
+                "cryptography==38.0.4" \
+                "awscli" \
+                --requirement reqs/requirements-dev-all-contrib-expectations.txt \
+                --editable ".[spark, sqlalchemy]" \
+                --constraint constraints-dev.txt
+            displayName: 'Install dependencies'
+
+          - bash: bash ./download_json_from_s3.sh
             workingDirectory: $(Build.SourcesDirectory)/assets/scripts/
-            displayName: 'Show implemented engines'
+            displayName: 'Copy JSON files from S3'
+            env:
+              # AWS credentials
+              AWS_ACCESS_KEY_ID: $(AWS_ACCESS_KEY_ID)
+              AWS_SECRET_ACCESS_KEY: $(AWS_SECRET_ACCESS_KEY)
+              AWS_DEFAULT_REGION: $(AWS_DEFAULT_REGION)
 
-          - bash: cat checklists.txt
+          - bash: for fname in $(ls -1 *_full.json); do echo -e "\n\n==========\n$fname"; cat "$fname"; done
             workingDirectory: $(Build.SourcesDirectory)/assets/scripts/
-            displayName: 'Show full checklist summary'
+            displayName: 'cat *_full.json files in assets/scripts'
 
-          - bash: grep -E "(^expect|Completeness checklist|^ *[A-z]|^ *-|-----)" checklists.txt | grep -vE '(No validate_configuration|Using default validate_configuration|Has a full suite|Has passed a manual)'
+          - bash: python ./build_gallery.py --backends "sqlite" --only-combine --outfile-name "expectation_library_v2--staging.json" 2>&1 | tee output--build_gallery.txt ; grep -o "ERROR - (.*" output--build_gallery.txt | sort > testing-error-messages.txt ; grep -o "Expectation type.*" output--build_gallery.txt | sort > gallery-exp-types.txt ; touch gallery-tracebacks.txt
```

Comment:
Why does there need to be a backend specified here for the combine step?

Reply:
Ok - but if we are throwing out the test results, couldn't we just create the `ExpectationDiagnostics` objects directly rather than re-running tests? I.e. with an empty list for `backend_test_result_counts` for example.

Reply:
The test results aren't getting tossed out. This is the time that the [sqlite_full.json](https://superconductive-public.s3.us-east-2.amazonaws.com/static/gallery/sqlite_full.json) file is generated.

I don't think there is a way to get a diagnostics object without running tests on at least one backend.